### ISSUE TITLE
Cannot find SnappySnippet on the Chrome Extension WebStore

### ISSUE THREAD

**Issue by KartikSoneji** [kdzwinel/SnappySnippet#70]
The only WebStore link that I could find is broken:
[https://chrome.google.com/webstore/detail/snappysnippet/blfngdefapoapkcdibbdkigpeaffgcil/](https://chrome.google.com/webstore/detail/snappysnippet/blfngdefapoapkcdibbdkigpeaffgcil/)

**Comment by kdzwinel** [comment]
Right, sorry for the confusion, I pulled all of my extensions from web store (some context: https://twitter.com/kdzwinel/status/1213070289292210176 ). Please install from github directly. Thank you!

**Comment by KartikSoneji** [comment]
Oh. I understand your decision. Maybe it would be a good idea to add a section to the readme explaining this?  
Also, have you considered adding a simple update check to the extension that checks the last commit on GitHub so that people do not miss out on updates?

**Comment by KartikSoneji** [comment]
I am working on adding FireFox support and fixing the "Open in ' JsBin/CodePen/JsFiddle buttons. Will you submit the extension as a FireFox addon if FireFox is supported?

**Comment by kalligator** [comment]
The one I have installed from CWS is at v0.5 while the latest release here is v0.4. 
Could you perhaps update the release here as well?
edit - Ok it's here, I hadn't noticed because 0.4 is shown prominently as the Latest one.

**Comment by slim16165** [comment]
> Right, sorry for the confusion, I pulled all of my extensions from web store (some context: https://twitter.com/kdzwinel/status/1213070289292210176 ). Please install from github directly. Thank you!

Hi, but how do we install SnappySnippet directly from GitHub?
EDIT: I found out that it's really easy, I'm leaving this comment just to help people searching on google

**Comment by KartikSoneji** [comment]
This might be helpful, but take the warnings with a grain of salt. I have been using this on Windows for a long time and haven't run into any issues.

https://stackoverflow.com/questions/24577024/install-chrome-extension-not-in-the-store
